### PR TITLE
protect against non-integer array indexing

### LINKED ISSUE #1594: the cell atlas of human thymic development cxg dataset doesn't load properly
the cell atlas of human thymic development cxg dataset doesn't load properly

https://cellxgene.cziscience.com/d/cell_atlas_of_thymic_development-14.cxg/

Error:
```
instrument.ts:129 TypeError: r.col(...).summarizeCategorical is not a function
    at Tp (controlsHelpers.js:96)
    at em.createCategorySummary (index.js:108)
    at new em (index.js:34)
    at go (react-dom.production.min.js:134)
    at Pa (react-dom.production.min.js:176)
    at vc (react-dom.production.min.js:263)
    at ls (react-dom.production.min.js:246)
    at cs (react-dom.production.min.js:246)
    at Jc (react-dom.production.min.js:239)
    at react-dom.production.min.js:123
    at t.unstable_runWithPriority (scheduler.production.min.js:19)
    at Bi (react-dom.production.min.js:122)
    at qi (react-dom.production.min.js:123)
    at Wi (react-dom.production.min.js:122)
    at es (react-dom.production.min.js:240)
    at Object.notify (Subscription.js:19)
    at e.t.notifyNestedSubs (Subscription.js:92)
    at e.t.handleChangeWrapper (Subscription.js:97)
    at d (redux.js:221)
    at index.js:11
    at dispatch (redux.js:638)
    at index.js:84
    at async Promise.all (/d/cell_atlas_of_thymic_development-14.cxg/index 1)
    at /d/cell_atlas_of_thymic_development-14.cxg/async https:/cellxgene.cziscience.com/static/main-5c978cc95fd0bd949a4d.js:2
```

### PR DESCRIPTION
**Pull request by bkmartinjr**
There were several critical modules in Dataframe that assumed that Array bracket access with a non-integer number would return undefined.  This failed when a dataset contained a category named "sort".

TL;DR:  `[]["sort"] === Array.prototype.sort`.

Which leads to all kinds of potential for mischief and mayhem.

Fixes #1594

### CODE REVIEW

File: client/src/util/dataframe/labelIndex.js [hunk 1]
```diff
@@ -41,31 +41,29 @@ class IdentityInt32Index {
     return k;
   }
 
-  /* eslint-disable class-methods-use-this -- having these accessors as class methods allows for polymorphism */
```
Comment (seve) [inline client/src/util/dataframe/labelIndex.js:44]: Thanks for updating the disable!
